index.js

```diff
@@ -36,6 +36,22 @@ const eslintConfig = {
   rules: {
     ...reactRules,
   },
+  overrides: [
+    {
+      // temporary override until everybody removes the React import
```

Comment:
We could remove this override in the next major. Or just leave it there; it's harmless.